Core/Analyzers/ClassRefAnalyzer.cpp

```diff
@@ -0,0 +1,22 @@
+#include "ClassRefAnalyzer.h"
+
+
+using namespace ObjectiveNinja;
+
+ClassRefAnalyzer::ClassRefAnalyzer(SharedAnalysisInfo info, SharedAbstractFile file)
+    : Analyzer(std::move(info), std::move(file))
+{
+}
+
+void ClassRefAnalyzer::run()
+{
+    const auto sectionStart = m_file->sectionStart("__objc_classrefs");
+    const auto sectionEnd = m_file->sectionEnd("__objc_classrefs");
+    if (sectionStart == 0 || sectionEnd == 0)
+        return;
+
+    // TODO: Dynamic Address size for armv7
```

Comment:
Does this need to be resolved prior to merging?

Reply:
No, it requires a later larger refactor to several portions of the tool, and this is just making a note at one of the many places that will have to be updated.